app/femr/ui/views/triage/index.scala.html

```diff
@@ -263,6 +250,19 @@ <h2 class="text-center">Check In - Triage</h2>
                     <input type="hidden" name="isDiabetesScreenPerformed" value="false" />
                 </div>
                 }
+        <div class="submitResetWrap">
+            <button type="submit" id="triageSubmitBtn" class="fButton fRedButton fSubmitButton pull-right">
+                Submit Patient</button>
+            <button type="button" class="fButton fOtherButton newPatientBtn pull-left">Reset Fields</button>
+            @if(viewModel.getPatient.getId != 0) {
+                @helper.form(action = SearchController.handleSearch("history")) {
+                    <input type="text" class="hidden" name="patientSearchQuery" value="@viewModel.getPatient.getId"/>
+                    <button type="submit" id="historyButton" class="fButton fOtherButton pull-left">Patient History</button>
+                }
+            }
+
```

Comment:
I think the anchor tag is a much more elegant solution.

@kdunlap are you talking about nested forms or multiple forms on one page?